### PR TITLE
Use ubuntu-latest instead of outdated centos-8

### PR DESCRIPTION
**Pull request by laeubi**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -18,7 +18,7 @@
 def runOnNativeBuildAgent(String platform, Closure body) {
 	def final nativeBuildStageName = 'Build SWT-native binaries'
 	if (platform == 'gtk.linux.x86_64') {
-		podTemplate(inheritFrom: 'centos-latest' /* inhert general configuration */, containers: [
+		podTemplate(inheritFrom: 'ubuntu-latest' /* inhert general configuration */, containers: [
 			containerTemplate(name: 'swtbuild', image: 'eclipse/platformreleng-centos-swt-build:8',
```
Comment (akurtakov) [inline Jenkinsfile:22]: What will the effect be of using eclipse/platformreleng-centos-swt-build:8 as containerTemplate?

Comment (laeubi) [reply Jenkinsfile:22]: I'm not sure what the question is, according to the doc we only inherit the general config and then run inside container (what is `eclipse/platformreleng-centos-swt-build:8`) maybe @HannesWell  can tell more?

Comment (laeubi) [reply Jenkinsfile:22]: Yes, the goal here is just to update the runtime to be prepared for new glibc

Comment (akurtakov) [reply Jenkinsfile:22]: OK, if it works, let's merge it and retry gtk pr.

**Comment by laeubi** [comment]
> OK, if it works, let's merge it and retry gtk pr.

I assume test failures are expected?!?

**Comment by akurtakov** [comment]
> > OK, if it works, let's merge it and retry gtk pr.
> 
> I assume test failures are expected?!?

If we speak about the 3 long failing Browser tests - yes.

**Comment by laeubi** [comment]
Yep

grafik

File: Jenkinsfile (hunk 1 above)
Comment (HannesWell) [reply Jenkinsfile:22]: > according to the doc we only inherit the general config and then run inside container (what is `eclipse/platformreleng-centos-swt-build:8`) maybe @HannesWell can tell more?

Exactly. Probably even the basic image is sufficient.
Let's find this out:
- https://github.com/eclipse-platform/eclipse.platform.swt/pull/1627

**Pull request merged**